### PR TITLE
modify submission.json format; will contain machine information in submission.json

### PR DESCRIPTION
**Pull request by felix5572**
modify submission.json format; will contain machine information in submission.json

### CODE REVIEW

**Review by njzjz — APPROVED** [review]

File: dpdispatcher/ssh_context.py [hunk 1]
```diff
@@ -261,6 +261,12 @@ def bind_submission(self, submission):
         # self.remote_root = os.path.join(self.temp_remote_root, self.submission.submission_hash, self.submission.work_base )
         self.remote_root = pathlib.PurePath(os.path.join(self.temp_remote_root, self.submission.submission_hash)).as_posix()
 
+        try:
+            sftp = self.ssh_session.ssh.open_sftp()
+            sftp.mkdir(self.remote_root)
+        except OSError:
```
Comment (njzjz) [inline dpdispatcher/ssh_context.py:267]: Is `OSError` for line 265 or line 266?

Comment (felix5572) [reply dpdispatcher/ssh_context.py:267]: 266 line. I will fix it.

**Review by njzjz — APPROVED** [review]

**Comment by njzjz** [comment]
@felix5572 we could use squash and merge in the future so you may need to open a new branch when submitting PRs.

File: dpdispatcher/ssh_context.py (hunk 1 above)
Comment (njzjz) [reply dpdispatcher/ssh_context.py:267]: I don't remember - why do we need to create a directory here?

**Pull request merged**